src/index.html

```diff
@@ -11,5 +11,16 @@
     <header class="app-header">
       <h1 class="app-header__heading">Quiz App</h1>
     </header>
+    <section class="quiz-card">
+      <h2 class="quiz-card__heading">Question</h2>
+      <p class="quiz-card__question">Here is the question</p>
+      <button class="quiz-card__button">Show answer</button>
+      <ul class="quiz-card__list">
+        <li class="quiz-card__tag">tag 1</li>
+        <li class="quiz-card__tag">tag 2</li>
+        <li class="quiz-card__tag">tag 3</li>
+        <li class="quiz-card__tag">tag 4</li>
+      </ul>
+    </section>
```

Comment:
The main content of a website should be wrapped in a `main` element. And to see whether something looks real or not, I would recommend you adding a real question and real tags 👍 Especially with the tags it might look very different depending on how long the tag name is.
```suggestion
   </main>
    <section class="quiz-card">
      <h2 class="quiz-card__heading">Question</h2>
      <p class="quiz-card__question">Here is the question</p>
      <button class="quiz-card__button">Show answer</button>
      <ul class="quiz-card__list">
        <li class="quiz-card__tag">tag 1</li>
        <li class="quiz-card__tag">tag 2</li>
        <li class="quiz-card__tag">tag 3</li>
        <li class="quiz-card__tag">tag 4</li>
      </ul>
    </section>
  </main>
```